### ISSUE TITLE
new specification

### ISSUE THREAD

**Issue by mytobygithub** [IversityMobileTeam/tweet-the-weather#37]
Hello everyone,

last week we got a new information:

https://iversity.org/my/courses/web-engineering-ii-developing-mobile-html5-apps/assessment_units/15205/intro

or here is the test of it:

Dear Students,

You still have more than two weeks to submit your app in the context of the final exam.

Please take into account that your submission must meet the following requirements in order to be reviewed:
1. Your app must be published in the cloud of cloudControl.
2. Your app must have been built for production with Sencha Cmd.
3. Your full name and contact information and, if you summit a team app, the full names and contact information of all team members, must be stated on a separate tab "Info".
4. Your app must visualise dynamic data from one or more RESTful Web APIs.
   At the beginning of next week iversity will announce the exact procedure how to register for a certificate of accomplishment. I guess there will be great news for all of us.

Keep on designing and coding your killer apps!

KQC

Until now, we did not really consider something for #3, has some a good idea?

Thanks tobi

**Comment by andrIvash** [comment]
Hello everyone. Do we have enough time to continue the project. I have seen the decreasing of activity ?

**Comment by arin-banerjee** [comment]
Yes, we have run out of time but may be we need an urgent group discussion
on gitter. May be a short discussion but if everyone is present then we can
sort out some solution.

I think the project is worth completing at this stage.

Please share your views so that we can fix a suitable time for discussion
if required.

Thanks,

**Comment by jviotti** [comment]
Hi all,

I'm sorry I've been quite busy this time but I can still take time to work on the project. I think we made quite a lot of progress so it's worth to finish what we started.

I've been assisting @mytobygithub into creating the Main View. There is quite a lot of work there so if anyone wants to help, please contact @mytobygithub.

I'll be in Gitter today if you want to have a chat!

Cheers!
